modules/audio.md

```diff
@@ -0,0 +1,124 @@
+{::options parse_block_html="true" /}
+
+<div class="row">
+<div class="large-12 columns">
+
+## <img class="constrain-sm" src="https://s3.amazonaws.com/technicalmachine-assets/technical-io/modules/usb.png"> Audio
+
+[<i class="fa fa-github">View source on Github</i>](https://github.com/tessel/tessel-av)
+
+### Step 1
+
+Make a directory inside your "tessel-code" folder called "audio", change directory into that folder, and initialize a tessel project:
+
+`mkdir audio; cd audio; t2 init`
+
+Download [this mp3](https://dl.dropboxusercontent.com/u/3531958/yoda-mudhole.mp3) into the "audio" directory: 
+
+`wget https://dl.dropboxusercontent.com/u/3531958/yoda-mudhole.mp3`
```

Comment:
Ok, so I guess the easiest thing would be to just say "Download this thing and put it your `audio` directory". Also, this file probably needs a different home.